### PR TITLE
add function readENERDATA.R

### PR DESCRIPTION
**Pull request by Fsioutas**

### CODE REVIEW

File: R/readENERDATA.R [hunk 1]
```diff
@@ -0,0 +1,105 @@
+#' ENERDATA
+#'
+#' Read in an csv file and convert it to a magpie object
+#' The data has information about electricity capacity per country and per year
```
Comment (giannou) [inline R/readENERDATA.R:4]: the dataset does not contain only capacity data, it contains several types of data. suggested change: "The dataset contains several energy-related data types from ENERDATA for various countries and years."

**Review by giannou — CHANGES_REQUESTED** [review]

File: R/readENERDATA.R (hunk 1 above)
Comment (Fsioutas) [reply R/readENERDATA.R]: ok

File: R/readENERDATA.R [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+#'
+#' @param subtype The type is referring to the capacity of electricity and the
+#' data that should be read is from the csv file "open_prom_database_20_4_2023_1.csv"
```
[the remark this answers is not in the document]
Comment (Fsioutas) [reply R/readENERDATA.R]: ok

File: R/readENERDATA.R [hunk 3]
```diff
@@ -0,0 +1,105 @@
+#' ENERDATA
```
[the remark this answers is not in the document]
Comment (Fsioutas) [reply R/readENERDATA.R]: ok

**Pull request merged**